src/main/java/blackjack/domain/player/AbstractPlayer.java

```diff
@@ -0,0 +1,60 @@
+package blackjack.domain.player;
+
+import blackjack.domain.card.PlayingCard;
+import blackjack.domain.card.PlayingCards;
+
+public abstract class AbstractPlayer implements Player {
+
+    protected PlayingCards playingCards;
+    protected String name;
+
+    @Override
+    public void addCard(PlayingCard playingCard) {
+        playingCards.addCard(playingCard);
+    }
+
+    @Override
+    public boolean isBlackJack() {
```

Comment:
모든 구현 클래스가 동일한 로직을 수행하도록 강제해야 한다면 final을 붙여 오버라이드를 방지할 수 있어요!
```suggestion
    public final boolean isBlackJack() {
```

Reply:
감사합니다! 👍